src/main/java/performance/LocustOperations.java

```diff
@@ -23,8 +23,8 @@
 	private static final String NAMEOFREPORT = "performanceResults";
 	private static final Logger logger = LoggerFactory.getLogger(LocustOperations.class);
 	private static String masterFilePath = FileOperations.getInstance().getAbsolutePath(ConfigReader.getInstance().getLocustMasterFilePath());
-    private static String csvReportFilePath = FileOperations.getInstance().getAbsolutePath(ConfigReader.getInstance().getCsvReportFolderPath());
     private static String operatingSystem = System.getProperty("os.name").toLowerCase();
+    //private static String csvReportFilePath = FileOperations.getInstance().getAbsolutePath(ConfigReader.getInstance().getCsvReportFolderPath());
```

Comment:
Commented line?

Reply:
Yes ... I didn't know what to do ... I commented this to get it if in the feature is solved ... We can remove it if you want ...